server/config/config.go

```diff
@@ -428,9 +428,57 @@ type FleetConfig struct {
 	GeoIP            GeoIPConfig
 	Prometheus       PrometheusConfig
 	Packaging        PackagingConfig
+	MDM              MDMConfig
 	MDMApple         MDMAppleConfig `yaml:"mdm_apple"`
```

Comment:
I think we should remove the old config now, or at least write a TODO command to remove this field later.

Reply:
Yes, will do that once we have clarification of what we support in the new flags (https://github.com/fleetdm/fleet/pull/8730#discussion_r1029424500). I will only remove the equivalent fields of the old config, though (to minimize impact on dogfood), unless you think that all of it can be safely removed?

Reply:
> or at least write a TODO command to remove this field later

There's already a ticket for the cleanup tasks: https://github.com/fleetdm/fleet/issues/8701